stopfmt.py

```diff
@@ -44,12 +43,17 @@ def run(self, edit):
             self.view.unfold(folds)
 
 class FoldListener(sublime_plugin.EventListener):
-    def on_load(self, view):
+    def run_fold_check(self, view):
         if view.score_selector(0,'source.go') > 0:
             settings = sublime.load_settings("stopfmt.sublime-settings")
             if settings.get('auto_fold_go', False):
                 view.run_command('fold_short_ifs')
 
+    def on_load_async(self, view):
```

Comment:
check on async, also run on save!

Reply:
I think for load sync is actually better since async is meant for cases where if you edit the buffer in the mean time it doesn't matter, but in this case it'll break. It's also fast enough that sync is fine.

Less clear for save I think since that's an operation where latency matters more, but the same considerations with it being fast and any asynchronous edits breaking it apply.